exercises/simple-linked-list/linked_list.exs

```diff
@@ -0,0 +1,83 @@
+defmodule LinkedList do
+  @opaque t :: tuple()
+
+  @doc """
+  Construct a new LinkedList
+  """
+  @spec new() :: t
+  def new() do
+    # Your implementation here...
```

Comment:
We don't really put these comments in the functions to be implemented. For some reason many folks don't delete them when they get around to doing the implementations.